src/BusinessLogic/Domain/PromotionalWidgets/MiniWidgetMessagesProvider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SeQura\Core\BusinessLogic\Domain\PromotionalWidgets;
+
+/**
+ * Interface MiniWidgetMessagesProvider
+ *
+ * @package SeQura\Core\BusinessLogic\Domain\PromotionalWidget
+ */
+class MiniWidgetMessagesProvider
```

Comment:
Shouldn't this implement an interface and be registered in the bootstrap, so that integrations could extend it?